target/linux/ramips/dts/mt7621_tplink_re500-v1.dts

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_tplink_rexx0-v1.dtsi"
+
+/ {
+	compatible = "tplink,re500-v1", "mediatek,mt7621-soc";
+	model = "TP-Link RE500 v1";
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
```

Comment:
Since LEDs are defined in the DTS files, please also move the led-aliases there; label-mac-device may stay in the DTSI.

Reply:
done